### PR TITLE
Implement config.yml for Template chooser

### PR DESCRIPTION
**Pull request by Andre601**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [ ] Internal code
- [ ] Library interface (affecting end-user code) 
- [ ] Documentation
- [x] Other: Issue-system (template chooser) of GitHub 

Closes Issue: #1252 

## Description
This PR implements a `config.yml` within the `.github/ISSUE_TEMPLATE` directory.
Having this file allows for the addition of external links to sources like the wiki, javadoc, or releases.
It also allows disabling of the option to skip choosing a template.

Current Settings are as follows:
- Creating of blank issues is __disabled__
- Following Links where added:
  - Discord: https://discord.gg/0hMr4ce0tIk3pSjp
  - Javadoc: https://ci.dv8tion.net/job/JDA/javadoc/
  - Jenkins: https://ci.dv8tion.net/job/JDA/
  - Wiki: https://github.com/DV8FromTheWorld/JDA/wiki
  - Releases: https://github.com/DV8FromTheWorld/JDA/releases

### CODE REVIEW

**Comment by Andre601** [comment]
It was also suggested to implement a issue template for questions.
I could commit it towards this PR, if I know the general look of the template itself.

**Comment by DV8FromTheWorld** [comment]
This looks fine so far.

**Comment by Andre601** [comment]
I decided to also implement a Template for general questions as suggested by @DV8FromTheWorld 

I based it off the Feature request template.
Don't hesitate to suggest changes to it.

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+name: Question
+about: Use this Template to ask a question regarding JDA (Joining the Discord is recommendet)
```
Comment (averen) [inline .github/ISSUE_TEMPLATE/question.md:3]: There is a typo in this line. I also tried to clarify the meaning.

```suggestion
about: Use this template to ask a question regarding JDA (Joining the Discord first is highly recommended)
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+<!--
+  This is not the place to learn Java. Please refer to StackOverflow for your
+  general programming questions: https://stackoverflow.com/questions/tagged/java
+-->
+
+## Question
+
+<!--
+Ask your question here and provide as much information as possible.
```
Comment (averen) [inline .github/ISSUE_TEMPLATE/question.md:34]: The other notes have indentation.

```suggestion
  Ask your question here and provide as much information as possible.
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+-->
+
+## Example Code
+
+<!--
+Provide some example code regarding your question here.
+Make sure to put it in a code block.
```
Comment (averen) [inline .github/ISSUE_TEMPLATE/question.md:41]: The other notes have indentation.

```suggestion
  Provide some example code regarding your question here.
  Make sure to put it in a code block.
```

File: .github/ISSUE_TEMPLATE/question.md (hunk 2 above)
Comment (Andre601) [reply .github/ISSUE_TEMPLATE/question.md:34]: This is based on the feature request template which also doesn't have a indent here, so I left it away on purpose.

File: .github/ISSUE_TEMPLATE/question.md (hunk 3 above)
Comment (Andre601) [reply .github/ISSUE_TEMPLATE/question.md:41]: See previous reply for reason.

**Comment by Andre601** [comment]
@DV8FromTheWorld does the question template look good?

**Pull request merged**